src/server/scripts/EasternKingdoms/ShadowfangKeep/boss_apothecary_hummel.cpp

```diff
@@ -116,14 +118,19 @@ class boss_apothecary_hummel : public CreatureScript
         void EnterEvadeMode() override
         {
             summons.DespawnAll();
+            std::list<Creature*> valentineAdds;
```

Comment:
the instance spawns them, no?

if (Hummel)
...
hummel->AI()->JustSummoned(add);
...

Reply:
that way they'll register as his summons and just despawn on evade without the need for searching grid